examples/permissioned-token/test/happy-paths.js

```diff
@@ -307,4 +308,90 @@ describe('Permissioned Token Tests - Happy Paths', function () {
       intialElonHolding.amount + oldReserveAssetHolding.amount
     );
   });
+
+  it('should cease tokens from bob', () => {
+    // Opt-In to ASA
+    ctx.optInToASA(bob.address);
+
+    // Issue few tokens to sender
+    ctx.issue(asaReserve.account, bob, 150);
+    ctx.syncAccounts();
+
+    const toCeaseAmt = 120n;
+    const ceaseTxParams = [
+      {
+        type: types.TransactionType.CallNoOpSSC,
+        sign: types.SignType.SecretKey,
+        fromAccount: asaManager.account,
+        appID: ctx.controllerappID,
+        payFlags: { totalFee: 1000 },
+        appArgs: [FORCE_TRANSFER_APPARG],
+        foreignAssets: [ctx.assetIndex]
+      },
+      {
+        type: types.TransactionType.RevokeAsset,
+        sign: types.SignType.LogicSignature,
+        fromAccountAddr: ctx.lsig.address(),
+        recipient: asaReserve.address,
+        assetID: ctx.assetIndex,
+        revocationTarget: bob.address,
+        amount: toCeaseAmt,
+        lsig: ctx.lsig,
+        payFlags: { totalFee: 1000 }
+      },
+      {
+        type: types.TransactionType.TransferAlgo,
+        sign: types.SignType.SecretKey,
+        fromAccount: asaManager.account,
+        toAccountAddr: ctx.lsig.address(),
+        amountMicroAlgos: 1000,
+        payFlags: { totalFee: 1000 }
+      }
+    ];
+
+    const initialBobBalance = ctx.getAssetHolding(bob.address).amount;
+    assert.equal(initialBobBalance, 150n);
+
+    const initialReserveBalance = ctx.getAssetHolding(asaReserve.address).amount;
+    ctx.runtime.executeTx(ceaseTxParams);
+
+    // verify cease
```

Comment:
```suggestion
    // verify toCeaseAmt amount is ceased
```

Reply:
`verify toCeaseAmt amount is ceased` sounds a bit cryptic

Reply:
then please change it to `verify cease amount`